adcc/backends/test_psi4.py

```diff
@@ -116,3 +116,18 @@ def template_rhf_h2o(self, basis):
         mints = psi4.core.MintsHelper(wfn.basisset())
         ao_dip = [np.array(comp) for comp in mints.ao_dipole()]
         operator_import_test(wfn, ao_dip)
+
+    def template_uhf_h2o(self, basis):
```

Comment:
I think it makes more sense to use `ch2nh2` here now ... same for PySCF, don't you agree?

Reply:
resolved.